server/config/mq-config.js

```diff
@@ -1,43 +1,37 @@
 const joi = require('@hapi/joi')
 
+const queueSchema = joi.object({
+  name: joi.string().required(),
+  endpoint: joi.string().required(),
+  queueUrl: joi.string().required(),
+  region: joi.string().default('eu-west-2'),
+  accessKeyId: joi.string().optional(),
```

Comment:
With the current implementation the dev keys will be passed to SQS in the cluster.  The key provider chain should attach the web token first over the access keys but given the issues we've had this is a heads up to try removing them if we encounter any future issues.

Reply:
Not sure I understand, are you talking about the `accessKeyId` and `secretAccessKey`? 
You would get accessKeyId and secretAccessKey as undefined as no value will be set in the Helm chart for the env var `DEV_ACCESS_KEY` and `DEV_ACCESS_KEY_ID`.

It's not a great having to define optional settings that are only used in development.
It could be removed from the config schema entirely, but then it would be a bit hidden.

Reply:
They would be undefined but the property would exist.  It might not be an issue and not saying this approach is wrong, more just something to keep in mind as a potential gotcha if we have issues with the credential chain.